packages/expo-cli/src/credentials/local/update.ts

```diff
@@ -0,0 +1,157 @@
+  }
+  if (['all', 'android'].includes(platform)) {
+    log('Updating Android credentials in credentials.json');
+    await updateAndroidAsync(ctx);
+  }
+  if (['all', 'ios'].includes(platform)) {
+    log('Updating iOS credentials in credentials.json');
+    await updateIosAsync(ctx);
+  }
+}
+
+async function updateAndroidAsync(ctx: Context) {
+  const credentialsJsonFilePath = path.join(ctx.projectDir, 'credentials.json');
+  let rawCredentialsJsonObject: any = {};
+  if (await fs.pathExists(credentialsJsonFilePath)) {
+    try {
+      const rawFile = await fs.readFile(credentialsJsonFilePath);
+      rawCredentialsJsonObject = JSON.parse(rawFile.toString());
+    } catch (error) {
+      log.error(`There was an error while reading credentials.json [${error}]`);
+      log.error('Make sure that file is correct (or remove it) and rerun this command.');
+      throw error;
+    }
+  }
+  const experienceName = `@${ctx.manifest.owner || ctx.user.username}/${ctx.manifest.slug}`;
+  const keystore = await ctx.android.fetchKeystore(experienceName);
+  if (!keystore) {
+    throw new Error('There are no credentials configured for this project on Expo servers');
+  }
+
+  const isKeystoreComplete =
+    keystore.keystore && keystore.keystorePassword && keystore.keyPassword && keystore.keyAlias;
+
+  if (!isKeystoreComplete) {
+    const { confirm } = await prompts({
+      type: 'confirm',
+      name: 'confirm',
+      message:
+        'Credentials on Expo servers might be invalid or incomplete. Are you sure you want to continue?',
+    });
+    if (!confirm) {
+      log.warn('Aborting...');
+      return;
+    }
+  }
+
+  const keystorePath =
+    rawCredentialsJsonObject?.android?.keystorePath ?? './android/keystores/keystore.jks';
+  await _updateFileAsync(ctx.projectDir, keystorePath, keystore.keystore);
+
+  rawCredentialsJsonObject.android = {
+    keystore: {
+      keystorePath,
+      keystorePassword: keystore.keystorePassword,
+      keyAlias: keystore.keyAlias,
+      keyPassword: keystore.keyPassword,
+    },
+  };
+  await fs.writeJson(credentialsJsonFilePath, rawCredentialsJsonObject, {
+    spaces: 2,
+  });
+}
+
+async function updateIosAsync(ctx: Context) {
+  const credentialsJsonFilePath = path.join(ctx.projectDir, 'credentials.json');
+  let rawCredentialsJsonObject: any = {};
+  if (await fs.pathExists(credentialsJsonFilePath)) {
+    try {
+      const rawFile = await fs.readFile(credentialsJsonFilePath);
+      rawCredentialsJsonObject = JSON.parse(rawFile.toString());
+    } catch (error) {
+      log.error(`There was an error while reading credentials.json [${error}]`);
+      log.error('Make sure that file is correct (or remove it) and rerun this command.');
+      throw error;
+    }
+  }
+
+  const bundleIdentifier = ctx.manifest.ios?.bundleIdentifier;
+  if (!bundleIdentifier) {
+    throw new Error('"expo.ios.bundleIdentifier" field is required in your app.json');
+  }
```

Comment:
This logic is not valid anymore - check out the getBundleIdentifier function is iOSBuilder - https://github.com/expo/expo-cli/blob/master/packages/expo-cli/src/commands/eas-build/build/builders/iOSBuilder.ts#L178
We probably want to move it to utils dir and import here.